Chat/Chat.cpp

```diff
@@ -114,6 +114,19 @@ void Chat::PrintSomeoneList(std::vector<std::string>& list) const
     chat_mutex_.Unlock();
 }
 
+void Chat::PrintMyList(std::vector<File>& list) const
+{
+    chat_mutex_.Lock();
+    cout << endl;
```

Comment:
Well that will work for console good. But if you will need to get it from UI or to write it to file?
Please, make sure that all outputs just returns values and move writing data from on top of functional. (To different clients of your dll could print them as they need.